build/ci/github-actions/index.md

```diff
@@ -28,6 +28,8 @@ The following GitHub Actions are available:
   builds.
 - [Docker Buildx Bake](https://github.com/marketplace/actions/docker-buildx-bake){:target="blank" rel="noopener" class=""}:
   enables using high-level builds with [Bake](../../bake/index.md).
+- [Docker Scout](https://github.com/docker/scout-action){:target="blank" rel="noopener" class=""}:
+  analyze Docker images for security vulnerabilities.
```

Comment:
Would be better to have this published to the GitHub Marketplace like other actions instead of repo link for better discoverability.

Reply:
It be there in the time but it isn't there now.